Automattic/wp-calypso
# Help Center: Handle closed chats

<img width="407" alt="Screenshot 2024-11-18 at 11 37 43" src="https://github.com/user-attachments/assets/9bae2567-03fd-4c57-9a22-42a02a3d2e41">


Fixes https://github.com/Automattic/dotcom-forge/issues/9775

Related patch:
D166397-code

## Proposed Changes

* Add footer for closed conversations

## Why are these changes being made?


* Closed interactions should be marked and treated differently

## Testing Instructions



* Checkout the related patch
* Checkout this PR
* Using the `help-center-experience` flag open the support history
* Start a chat with support
* Find the relevant ticket in staging, and mark it as solved
* Open the support history again and locate and open the chat
* The footer for closed interactions should display
* Clicking on the button should create a new chat
